### PR TITLE
Adding HZAU

### PR DESCRIPTION
**Pull request by RimuroSuraimu**
Content created for review. 

Content quality is limited as information submitted is limited for review.

### CODE REVIEW

**Comment by RimuroSuraimu** [comment]
Resovlved #44

File: docs/campus/HZAU.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: 华中农业大学
+author: 蔡赵天
+date: 2023-03-31 00:00
+---
+
+### 概况
```
Comment (Cryolitia) [inline docs/campus/HZAU.md:7]: top-level 标题请使用 ``##``

File: docs/campus/HZAU.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+学校位于 [湖北省武汉市洪山区狮子山街1号](https://amap.com/place/B001B08XU8)，单体校园，面积很大，但教学区之间紧邻。
+
+
+### 军训与体育
+
+军训要求剪短头发，但存在沟通后保留头发的先例。
+
+> 军训被辅导员要求剪头发, 提供相关诊断证明后交涉成功可以保留。第一学期的体测也这样免了，之后还有次体测因二次术后挂了半年也没参加，校医院开的证明。
+
+学校有早操要求， 大一大二住宿舍的必须出操，有人查宿舍。
```
Comment (Cryolitia) [inline docs/campus/HZAU.md:25]: 建议添加换行
```suggestion
学校有早操要求， 大一大二住宿舍的必须出操，有人查宿舍。

同时有体育课的学期环湖跑计入，不达圈数即挂科。
```

File: docs/campus/HZAU.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### LGBT 友善梗概
+学校没有处理跨性别相关经验， 但是更改身份证后学校虽有迟疑的态度但仍然正常更改学籍。 
+
+> 复学后学籍等部门在我之前是从没遇到过的，学院领导也是满脸狐疑，毕竟是第一次。笔者这个情况，前无古人；后有没有来者，还不知道。
+
+
+### 投稿人
+
+蔡赵天 beta_cai@webmail.hzau.edu.cn
```
Comment (Cryolitia) [inline docs/campus/HZAU.md:45]:
```suggestion
蔡赵天 <beta_cai@webmail.hzau.edu.cn>
```

File: docs/campus/HZAU.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: 华中农业大学
+author: 蔡赵天
+date: 2023-03-31 00:00
+---
+
+### 概况
+
+> 规模大概能有几十个人，但很多人并没有在各种 QQ 群和微信群里紧密联系。由于有 purple 这个性少数社团，所以很容易能和一群友好的人们建立联系。总体环境不错。
```
Comment (Cryolitia) [inline docs/campus/HZAU.md:9]: 奇奇怪怪的一句话……

**Review by Cryolitia — CHANGES_REQUESTED** [review]
辛苦了喵

**Comment by Rinfair-CSP-A016** [comment]
修改意见同上

**Review by Cryolitia — APPROVED** [review]

**Pull request merged**